pkg/url/url.go

```diff
@@ -0,0 +1,14 @@
+package url
+
+import (
+	"net/url"
+)
+
+func IsAbsolute(raw string) bool {
+	u, err := url.Parse(raw)
+	if err != nil || u.Scheme == "" || u.Host == "" {
+		return false
```

Comment:
I'm not sure about this... I at least wanted to perform a basic check to see if a URL is valid according to a 'human' definition of a "correct" URL (of course, many incorrect ones would still pass). 

Or it might be better to use some existing validator...

Reply:
Yes, I think it can be done using the `validator` package.

Reply:
Thanks, I am going to change it in separate PR once this one is merged 👍